mdoc/Mono.Documentation/Util/CecilExtensions.cs

```diff
@@ -86,7 +86,8 @@ public static IEnumerable<TypeDefinition> GetTypes (this AssemblyDefinition asse
                                         .Select (et => et.Resolve ())
                                         .Where(e => e != null);
 
-            var types = assembly.Modules.SelectMany (md => md.GetAllTypes ()).Union(exportedTypes);
+            var types = assembly.Modules.SelectMany (md => md.GetAllTypes ()).Union
+            (exportedTypes.Where(t => assembly.MainModule.ExportedTypes.ToList().Exists(s => s.FullName == t.FullName)));
```

Comment:
I'm not sure this is going to do anything ... the `exportedTypes` variable is already coming from `assembly.MainModule.ExportedTypes`, so comparing it against that same list really shouldn't do anything.